tools/provision.ps1

```diff
@@ -306,16 +306,20 @@ function Install-PipPackage {
     Exit -1
   }
   Write-Host " => Attempting to install Python packages" -foregroundcolor DarkYellow
-  $pythonPath = [Environment]::GetEnvironmentVariable("OSQUERY_PYTHON_PATH", "Machine")
```

Comment:
Hrm, this might be an issue as we make use of this [env variable in the code base](https://github.com/facebook/osquery/blob/f89392bdb447e053779268ef4cb964eb88003b1e/osquery/core/windows/process.cpp#L321), though it's not used a lot. We should maybe add some logic here to figure out what the Python path is from the [system registry](https://stackoverflow.com/questions/647515/how-can-i-get-python-path-under-windows).

Reply:
Ups my bad. I see where it is being used and we could remove that but maybe the best is if I keep `OSQUERY_PYTHON_PATH` so that we have more control over the python version we're using after verifying that here. What do you think?